### ISSUE TITLE
508-defect-2 [AXE-CORE]: Heading levels SHOULD increase by one

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8556]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Heading orders **should** follow semantically correct order, increasing by one on the page. The "Need Help?" heading is an `h5` but is only preceded by `h1`, `h2`, and `h3`. 

For this use case, there is a need help component that should be used (see [Potential fix](#potential-fixes)).

This is also related to another issue **TICKET # TO COME**.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the hierarchy of the page content using heading levels to save time and frustration.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination: staging & production

### Current code
```html
  <footer class="need-help-footer row vads-u-padding-x--1p5">
    <div style="max-width: 600px;">
      <h4>Need Help?</h4>
      <hr>
      <p>You can call the VA Caregiver Support Line at <a href="tel:18552603274">855-260-3274</a>. We’re here Monday through Friday, 8:00 a.m. to 8:00 p.m. ET.</p>
      <p>You can also call our main VA information line at <a href="tel:18772228387">877-222-8387</a>, or contact your local Caregiver Support Coordinator.</p>
      <span><a target="_blank" rel="noopener noreferrer" href="https://www.caregiver.va.gov/support/New_CSC_Page.asp">Use our online Caregiver Support Coordinator search tool</a></span>
      <p>If this form isn't working right for you, please call us at at <a href="tel:18555747286">1-855-574-7286</a> <span>If you have hearing loss, call TTY: 711</span></p>
    </div>
  </footer>
```

### The component that should be used here is:
```html
  <div class="usa-width-two-thirds medium-8 columns">
    <div class="help-footer-box">
      <h2 class="help-heading">Need help?</h2>
      <div>
        <div>
          <p class="help-talk">
            For help filling out this form, or if the form isn’t working right, please <span>call VA Benefits and Services at <a href="tel:18008271000">800-827-1000</a>.<br> If you have hearing loss, call TTY: 711.</span>
          </p>
        </div>
      </div>
    </div>
  </div>
```

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 - Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.4/heading-order)

### Current h5
![h5](https://user-images.githubusercontent.com/57469/80607407-745cdf80-8a03-11ea-9f5c-da4ff018adf5.png)

### Changed to an h4
![h4](https://user-images.githubusercontent.com/57469/80607437-7e7ede00-8a03-11ea-8e81-915260e85216.png)

**Comment by jonathanjnelson** [comment]
Addressed in [ticket ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9342)

**Comment by jenstrickland** [comment]
The heading and markup for the Need Help? component is different in this form that elsewhere in the site. Please see the Possible Fixes section for the expected code structure and styling.

**Comment by LinnJS** [comment]
@jenstrickland as stated in the comment above this one was defiantly related to the landmark issue, and is resolved as well.

**Comment by jenstrickland** [comment]
Validated! 🎉
